frontend/app/transforms/django-duration.js

```diff
@@ -42,7 +42,7 @@ export default class DjangoDurationTransform extends Transform {
    */
   serialize(deserialized) {
     if (!moment.isDuration(deserialized)) {
-      return null;
+      return deserialized;
```

Comment:
Maybe throw a warning that a non-duration was received? Might help debugging.

Reply:
Done